### DISCUSSION TITLE
Use `[]` as the default value for a "Tags" field

### DISCUSSION THREAD

**Discussion by AaronBeaudoin** [directus/directus#12887] [Feature Requests]
Currently, when you create a "Tags" field, Directus does not set anything for the "Default Value" option.

![image](https://user-images.githubusercontent.com/26592486/164284734-8ed1363e-6822-4c2c-bc3c-c22f4cc01178.png)

It looks like this causes the value to end up being `""`, due to some Directus behavior.

![image](https://user-images.githubusercontent.com/26592486/164284873-71166daa-ad4b-4572-8835-11f553c4e056.png)

When you edit the field and remove `""` from the default value, it gets put back. It looks like Directus validates that the default value is valid JSON, and if not it just puts the `""` there.

Therefore the value can't really be `NULL` since Directus won't let you do that. So the default value initially is a bit misleading.

Also, when you add a tag and save the item, then remove the tag and save it again, the value of the field will now just be an empty array `[]`, so I think it makes much more sense to use `[]` as the default value instead of `""`.

**Comment by rijkvanzanten** [comment]
@azrikahar Could this be a side effect of rendering `null` as an attempted string from https://github.com/directus/directus/pull/12703?

**Reply by azrikahar** [reply]
oddly enough even before that PR, the value always is an empty string rather than `null`, so the PR technically is working _more correctly_ in a sense. Turns out it's just that field-detail-advanced-schema is passing the empty string all along! 😄

https://github.com/directus/directus/blob/59eb0f7cec93a373c5d2ad8ec80942c48e0837e9/app/src/modules/settings/routes/data-model/field-detail/field-detail-advanced/field-detail-advanced-schema.vue#L113

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
